server/server.go

```diff
@@ -76,6 +76,9 @@ const (
 
 var systemMainPath = ast.MustParseRef("data.system.main")
 
+// map of unsafe buitins
+var unsafeBuiltinsMap = map[string]bool{"http.send": true}
```

Comment:
Can we make this reference the AST declaration some how? It would be nice to catch inconsistency issues at compile time if possible.

Reply:
Done